src/edu/cornell/mannlib/vitro/webapp/edit/n3editing/configuration/generators/AddProjectUpdateGenerator.java

```diff
@@ -315,4 +308,46 @@ public EditMode getEditMode(VitroRequest vreq) {
         return editMode;
     }
 
```

Comment:
Would it be possible to put all of these queries into files that are loaded from the webapp resources?